gem5-configs/configs-boot-tests/system/MESI_Two_Level.py

```diff
@@ -162,7 +162,7 @@ def __init__(self, system, ruby_system, cpu, num_l2Caches):
                             is_icache = False)
         self.l2_select_num_bits = int(math.log(num_l2Caches , 2))
         self.clk_domain = cpu.clk_domain
-        self.prefetcher = RubyPrefetcher.Prefetcher()
+        self.prefetcher = RubyPrefetcher()
```

Comment:
Does this work with my patch? We need to be careful that this isn't out of sync with the staging branch.

Reply:
This works with your patch + the other patch that I uploaded on Teams.